zammad/README.md

```diff
@@ -1,48 +1,93 @@
 # Zammad Helm Chart
 
-This directory contains a Kubernetes chart to deploy [Zammad](https://zammad.org/) ticket system.
+A [Helm](https://helm.sh) chart to install [Zammad](https://zammad.org) on [Kubernetes](https://kubernetes.io)
 
-## Prerequisites Details
+[Zammad](https://zammad.org/) is a web based open source helpdesk/customer support system with many
+features to manage customer communication via several channels like telephone,
+facebook, twitter, chat and emails. It is distributed under version 3 of the
+GNU AFFERO General Public License (GNU AGPLv3).
 
-- Kubernetes 1.19+
-- Helm 3.x
-- Cluster with at least 4GB of free RAM
-
-## Chart Details
+## Introduction
 
 This chart will do the following:
 
-- Install Zammad statefulset
-- Install Elasticsearch, Memcached, PostgreSQL & Redis as requirements
+- Install a Zammad `StatefulSet`
+- Install Elasticsearch, Memcached, PostgreSQL, Redis & Minio (optional) as requirements
+
+Be aware that the Zammad Helm chart version is different from the actual Zammad version.
+
+## Prerequisites
+
+- Kubernetes 1.19+
+- Helm 3.2.0+
+- Cluster with at least 4GB of free RAM
 
 ## Installing the Chart
 
 To install the chart use the following:
 
 ```console
 helm repo add zammad https://zammad.github.io/zammad-helm
-helm upgrade --install zammad zammad/zammad --namespace zammad
+helm upgrade --install zammad zammad/zammad
```

Comment:
I dropped the namespace for simplicity. Otherwise the command fails unless the namespace is created before.